roles/nuage_master/tasks/serviceaccount.yml

```diff
@@ -1,42 +1,21 @@
 ---
-- name: Create temporary directory for admin kubeconfig
-  command: mktemp -u /tmp/openshift-ansible-XXXXXXX.kubeconfig
-  register: nuage_tmp_conf_mktemp
-  changed_when: False
-  run_once: True
-  delegate_to: "{{ nuage_ca_master }}"
-
-- set_fact:
-    nuage_tmp_conf: "{{ nuage_tmp_conf_mktemp.stdout }}"
-  run_once: True
-  delegate_to: "{{ nuage_ca_master }}"
-
-- name: Copy Configuration to temporary conf
-  command: >
-    cp {{ openshift.common.config_base }}/master/admin.kubeconfig {{nuage_tmp_conf}}
-  changed_when: false
-  run_once: True
-  delegate_to: "{{ nuage_ca_master }}"
-
 - name: Create Admin Service Account
   oc_serviceaccount:
-    kubeconfig: "{{ openshift_master_config_dir }}/admin.kubeconfig"
     name: nuage
     namespace: default
     state: present
   run_once: True
   delegate_to: "{{ nuage_ca_master }}"
 
 - name: Configure role/user permissions
-  command: >
-    {{ openshift.common.client_binary }} adm {{item}}
-    --config={{ nuage_tmp_conf }}
+  delegate_to: "{{ nuage_ca_master }}"
```

Comment:
Maybe put this at the end of the `Configure role/user permissions` so it aligns with the remaining tasks.